src/components/Portal/Portal.tsx

```diff
@@ -5,9 +5,15 @@ import {usePortalContainer} from '../utils/usePortalContainer';
 export interface PortalProps {
     container?: HTMLElement;
     children?: React.ReactNode;
+    disablePortal?: boolean;
 }
 
-export function Portal({container, children}: PortalProps) {
+export function Portal({container, children, disablePortal}: PortalProps) {
     const defaultContainer = usePortalContainer();
+
+    if (disablePortal) {
+        return <React.Fragment>{children}</React.Fragment>;
```

Comment:
Why wrap in fragment?

Reply:
Otherwise, when using `Portal`, you get ts error: `Portal` cannot be used as a JSX component. Its return type 'ReactNode' is not a valid JSX element.